packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/attach/telecom-telephony-alias-portabilit-portabilities-attach.controller.js

```diff
@@ -0,0 +1,51 @@
+import endsWith from 'lodash/endsWith';
```

Comment:
You should probably shorten your file names. Here we're in the `/telecom/telephony/alias/portability/attach` folder (or something like that). I think naming it `portabilities-attach.controller` is much more maintainable and readable